autoload/neomake/makers/ft/purescript.vim

```diff
@@ -0,0 +1,59 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#purescript#EnabledMakers()
+  return ['pulp']
+endfunction
+
+function! neomake#makers#ft#purescript#pulp()
+    return {
+        \ 'exe': 'pulp',
+        \ 'args': ['build', '--no-psa', '--', '--json-errors'],
```

Comment:
You're right, I've added a comment and a link to the issue where I got this information.